app/src/main/java/fr/free/nrw/commons/upload/worker/UploadWorker.kt

```diff
@@ -81,14 +81,13 @@ class UploadWorker(var appContext: Context, workerParams: WorkerParameters) :
 
 
     //Attributes of the current-upload notification
-    private var currentNotificationID: Int = -1// lateinit is not allowed with primitives
+    private var currentNotificationID: Int = -1// late init is not allowed with primitives
```

Comment:
Is this change intended?

Reply:
I made this change because there was a typo error in Android Studio. However, I didn't notice that this was about **`lateinit`** property. I'll change it if you want.

Reply:
Yes, please. 

Tip: Most of the codebases require refactoring after some time and IDEs show up various suggestions. Please feel free to open new issues as and when you find it relevant. This approach would also help in limiting the scope of any PR and discussing more about such suggestions independently as we would be able to break the huge task of refactoring into doable units :)

Reply:
Got it, Thanks :)